WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -181,9 +181,22 @@ private extension CommentDetailViewController {
     }
 
     func configureNavigationBar() {
-        if comment.canModerate {
-            navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(editButtonTapped))
-        }
+        // apply a Messages-like visual blur effect when content is scrolled.
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithTransparentBackground()
+        appearance.backgroundEffect = UIBlurEffect(style: .systemUltraThinMaterial)
+
+        // to apply visual blur only when content is scrolled, don't change the scrollEdgeAppearance.
+        self.navigationController?.navigationBar.isTranslucent = true
+        self.navigationItem.compactAppearance = appearance
+        self.navigationItem.standardAppearance = appearance
+
+        configureEditButtonItem()
+    }
+
+    func configureEditButtonItem() {
+        let editButtonItem = comment.canModerate ? UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(editButtonTapped)) : nil
+        navigationItem.rightBarButtonItem = editButtonItem
```

Comment:
Nitpick: This could just be one line.
`navigationItem.rightBarButtonItem = comment.canModerate ? UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(editButtonTapped)) : nil`

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17255/commits/7b5afc4ec613834804af270b21ac95b11f560d98. Thanks!